docs/caikit-servingruntime.yaml

```diff
@@ -31,7 +31,7 @@ data:
                   connection:
                     hostname: localhost:8033
 ---
-apiVersion: serving.kserve.io/v1alpha1
+apiVersion: serving.kserve.io/v1beta1
```

Comment:
I think this should remain v1alpha1, while inference service should have v1beta1

Reply:
alternatively, I guess we could remove the line at all and it grabs the one installed in the cluster, doesn't it?

Reply:
Yeah, let's match what KServe does today, or if we can omit it, that sounds like a good idea as well

Reply:
Got rid of `apiVersion` declarations in the last push

Reply:
sorry Daniele, i just re-tested it and it must have an api version.

Reply:
@bdattoma Updated again with the same format as the latest kserve docs. Let me know if it's fine.